plugins/node/opentelemetry-instrumentation-pg/README.md

```diff
@@ -0,0 +1,75 @@
+# OpenTelemetry Postgres Instrumentation for Node.js
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`pg`](https://github.com/brianc/node-postgres).
```

Comment:
i think we should mention that it also handle `pg-pool` here

Reply:
Both plugins had the same in their respective readme, The provided link is for mono repo which contains both `pg` and `pg-pool`. I guess this is correct though. I'm enlisting later pg and pg-pool separately too. I will change just a title of this link to have pg-pool too